microsoft-edge/web-platform/os-regional-settings.md

```diff
@@ -0,0 +1,96 @@
+---
+title: OS Regional Data Display in Microsoft Edge
+description: How users and web developers can use the OS regional format in Microsoft Edge for improved site experiences
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 08/15/2022
+--- 
+# Introducing the OS regional sharing setting
+
```

Comment:
Global:
- break up paragraphs
- reduce use of parentheses
- straighten curly quotes
- when referring to UI (vs. APIs), capitalize per the UI labels

Reply:
broke up paragraphs. reduced parenthesis, can't find curly quotes, checking with travis on his reviiew for UI versus APIs.